packages/transport-bridge/src/http.ts

```diff
@@ -154,6 +164,7 @@ export class TrezordNode {
                 parseBodyJSON,
                 (req, res) => {
                     res.setHeader('Content-Type', 'text/plain');
+                    // TODO: how to abort subscription?
```

Comment:
hmm imho aborting a single request should not clear subscriptions since they might have been created by another client